tests-behat/bootstrap/Context.php

```diff
@@ -59,10 +59,10 @@ protected function disableAnimations(): void
         };
 
         $css = $toCssFx('*', [
-            'animation-delay' => '0.02s',
-            'animation-duration' => '0.02s',
-            'transition-delay' => '0.02s',
-            'transition-duration' => '0.02s',
+            'animation-delay' => '0.005s',
```

Comment:
improve stability, needed for about modal.feature in about 1 of 20 tests

with this fix, it seems hundreds of tests can be run in a row without an issue